guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -88,14 +163,38 @@ Get the version number of desired version. The following example targets version
 
 In many cases, the exported archive content can be several gigabytes in size. You might want to split it smaller sizes or chunks. You can use the `--chunk-size-gb` option with in the `hammer content-export` command to handle this. The following example uses the `--chunk-size-gb=2` to split the archives into `2 GB` chunks.
 
+
 [options="nowrap" subs="+quotes"]
 ----
 # hammer content-export complete version --content-view=view --version=1.0 --organization=export-21527 --chunk-size-gb=2
 
 # ls -lh  /var/lib/pulp/exports/export-21527/view/1.0/2021-02-25T21-15-22-00-00/
 ----
 
+[[Using_ISS-Destination-Server]]
+=== Keeping track of your exports
+
+Consider the following scenario:
 
+You are exporting and importing content across several {ProjectServer}s.
+If you have to keep track of which server imports what content and what can and cannot be incrementally updated, things can get tedious.
+This is where the `--destination-server` option becomes useful.
```

Comment:
These lines can be streamlined (also didn't like the usage of tedious - that sets a negative image) to something like:

`When importing content to several {ProjectServer}s, the --destination-server option is especially useful for keeping track of which content was exported and to where.`

Reply:
/me needs to be more positive....